src/features/dsPreview/state/dsPreviewActions.ts

```diff
@@ -7,17 +7,17 @@ export function loadDsPreview(ref: QriRef): ApiActionThunk {
 
     try {
       await Promise.all([
-        dispatch(fetchDsPreview(ref)),
+        // dispatch(fetchDsPreview(ref)),
```

Comment:
hmmmm, is the idea here that because we now load previews to populate the dataset headers, not to fetch them again in the `loadDsPreview`?

If so, I would vote for adding back in `dispatch(fetchDsPreview(ref))`, or at least adding a TODO or some commentary. I would be confused in the future when using `loadDsPreview` that it doesn't actually load a bunch of the dataset.